app-conf/elephant.conf

```diff
@@ -27,7 +27,12 @@ enable_analytics=false
 
 # Additional Configuration
 # Check https://www.playframework.com/documentation/2.2.x/ProductionConfiguration
-jvm_props="-Devolutionplugin=enabled -DapplyEvolutions.default=true"
+#jvm_props="-Devolutionplugin=enabled -DapplyEvolutions.default=true"
+
+# Adding the below line for Heap Tuning and Java OPTS
+# mem - Heap Memory
+jvm_args="-Devolutionplugin=enabled -DapplyEvolutions.default=true -mem 1024 -J-Xloggc:$project_root../logs/elephant/dr-gc.`date +'%Y%m%d%H%M'` -J-XX:+PrintGCDetails"
```

Comment:
Is a slash missing here?

`$project_root/..`

Reply:
Nope Its not. `$project_root itself` has a `/` within it. So there is not point of mentioning one more `/`